src/components/App/filters/filters.js

```diff
@@ -68,3 +76,231 @@ export function formatVersion (string) {
+  let translationId = ''
+  switch (value) {
+    case ASSET_REQUEST_TYPES.createAsset: {
+      translationId = 'filters.asset-request-types-filter.create'
+      break
+    }
+    case ASSET_REQUEST_TYPES.updateAsset: {
+      translationId = 'filters.asset-request-types-filter.update'
+      break
+    }
+    default: {
+      translationId = 'filters.asset-request-types-filter.default'
+      break
+    }
+  }
+  return globalize(translationId)
+}
+
+export function assetPoliciesVerboseFilter (value) {
+  let translationId = ''
+  switch (value) {
+    case ASSET_POLICIES_VERBOSE[ base.xdr.AssetPolicy.transferable().value ]: {
+      translationId = 'filters.asset-policies-verbose-filter.transferable'
+      break
+    }
+    case ASSET_POLICIES_VERBOSE[ base.xdr.AssetPolicy.baseAsset().value ]: {
+      translationId = 'filters.asset-policies-verbose-filter.base-asset'
+      break
+    }
+    // eslint-disable-next-line max-len
+    case ASSET_POLICIES_VERBOSE[ base.xdr.AssetPolicy.statsQuoteAsset().value ]: {
+      translationId = 'filters.asset-policies-verbose-filter.stats-quote-asset'
+      break
+    }
+    case ASSET_POLICIES_VERBOSE[ base.xdr.AssetPolicy.withdrawable().value ]: {
+      translationId = 'filters.asset-policies-verbose-filter.withdrawable'
+      break
+    }
+    // eslint-disable-next-line max-len
+    case ASSET_POLICIES_VERBOSE[ base.xdr.AssetPolicy.issuanceManualReviewRequired().value ]: {
+      translationId = 'filters.asset-policies-verbose-filter.issuance-manual-review-required'
+      break
+    }
+    // eslint-disable-next-line max-len
+    case ASSET_POLICIES_VERBOSE[ base.xdr.AssetPolicy.canBeBaseInAtomicSwap().value ]: {
+      translationId = 'filters.asset-policies-verbose-filter.base-atomic-swap'
+      break
+    }
+    // eslint-disable-next-line max-len
+    case ASSET_POLICIES_VERBOSE[ base.xdr.AssetPolicy.canBeQuoteInAtomicSwap().value ]: {
+      translationId = 'filters.asset-policies-verbose-filter.quote-atomic-swap'
+      break
+    }
+    default: {
+      translationId = 'filters.asset-policies-verbose-filter.default'
+      break
+    }
+  }
+  return globalize(translationId)
+}
+export function stellarAssetTypesFilter (value) {
+  let translationId
+
+  switch (value) {
+    case STELLAR_TYPES.credit_alphanum4:
+      translationId = 'filters.stellar-asset-types-filter.alphanumeric-4'
+      break
+
+    case STELLAR_TYPES.credit_alphanum12:
+      translationId = 'filters.stellar-asset-types-filter.alphanumeric-12'
+      break
+
+    case STELLAR_TYPES.native:
+      translationId = 'filters.stellar-asset-types-filter.native'
+      break
+
+    default:
+      translationId = 'filters.stellar-asset-types-filter.default'
+      break
+  }
+
+  return translationId
+}
+
+export function limitsRequestStatesStrFilter (value) {
+  let translationId = ''
+  let key = Object.keys(LIMITS_REQUEST_STATES_STR)
+  switch (value) {
+    case key[0]: {
```

Comment:
How I can understand what value consists in key[0]

Reply:
done